gui/system/forms.py

```diff
@@ -153,6 +153,27 @@ def clean_path_locked(mp):
             )
 
 
+def clean_certificate(instance, certificate):
+    if not certificate:
+        raise forms.ValidationError(_("Empty Certificate!"))
+    nmatches = check_certificate(certificate)
+
+    if nmatches > 1:
+        instance.cert_chain = True
+
+    try:
+        load_certificate(certificate)
+    except crypto.Error:
+        raise forms.ValidationError(_("CA not in PEM format."))
+
+    #
+    # Should we validate the chain??? Probably
+    # For now, just assume the user knows WTF he is doing
```

Comment:
Please,  remove the WTF ,  it is not appropriate for an Open Source project!  And if user is doing a mistake we should handle that in a proper way.

Thanks!